src/Components/Web.JS/src/Rendering/RenderBatch/SharedMemoryRenderBatch.ts

```diff
@@ -107,7 +107,7 @@ const frameReader = {
   textContent: (frame: RenderTreeFrame) => platform.readStringField(frame as any, 16),
   markupContent: (frame: RenderTreeFrame) => platform.readStringField(frame as any, 16)!,
   attributeName: (frame: RenderTreeFrame) => platform.readStringField(frame as any, 16),
-  attributeValue: (frame: RenderTreeFrame) => platform.readStringField(frame as any, 24),
+  attributeValue: (frame: RenderTreeFrame) => platform.readStringField(frame as any, 24, true),
```

Comment:
Are there other places where this needs to be changed? Should we always be reading values with `readBoolValAsString` set to `true`?

Reply:
I suspect this is the only scenario for it. It is a pretty weird special case.